src/routes/docs/tutorials/sveltekit-csr-auth/step-3/+page.markdoc

```diff
@@ -21,7 +21,6 @@ import { Databases, Account, Client } from "appwrite";
 
 const client = new Client();
 client
-  .setEndpoint(PUBLIC_APPWRITE_ENDPOINT)
```

Comment:
Other parts of the tutorial needs to be updated too since this is removed, right?